### ISSUE TITLE
[1-Click Confetti] ... Confetti does not fire, only sound plays

### ISSUE THREAD

**Issue by BariHari** [raycast/extensions#14948]
### Extension

https://www.raycast.com/peduarte/1-click-confetti

### Raycast Version

1.84.1

### macOS Version

14.7

### Description

Confetti no longer fires, just the sound plays. ::sadcat emoji::

### Steps To Reproduce

I use an alias to launch the confetti extension. I have tried uninstalling and reinstalling it, but the issue persists. Confetti does not appear, regardless of which app is open.

**Comment by peduarte** [comment]
Still happening? Just tried and it's working here

**Comment by BariHari** [comment]
Yeah, I just checked. Maybe it's a setting I have turned on. It's not the end of the world, but the confetti does put a smile on my face lol

**Comment by Ryanjso** [comment]
Did you ever fix this I'm having the same issue?

**Comment by barihari** [comment]
> Did you ever fix this I'm having the same issue?

No never worked for me. It's still broken.

**Comment by Ryanjso** [comment]
I was able to fix it, i realized this command just calls the confetti tool in raycast, and I had that tool disabled
